### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by jkroll-deepgram** [deepgram/community#444] [General help]
Testing auto-reply...

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
